renderer/index.js

```diff
@@ -140,41 +120,25 @@ function lazyLoadCast () {
   return Cast
 }
 
-// Load the WebTorrent module, connect to both the WebTorrent and BitTorrent
-// networks, resume torrents, start monitoring torrent progress
+// Talk to WebTorrent process, resume torrents, start monitoring torrent progress
 function initWebtorrent () {
-  WebTorrent = require('webtorrent')
-
-  // Connect to the WebTorrent and BitTorrent networks. WebTorrent Desktop is a hybrid
-  // client, as explained here: https://webtorrent.io/faq
-  state.client = new WebTorrent()
-  state.client.on('warning', onWarning)
-  state.client.on('error', function (err) {
-    // TODO: WebTorrent should have semantic errors
-    if (err.message.startsWith('There is already a swarm')) {
-      onError(new Error('Couldn\'t add duplicate torrent'))
-    } else {
-      onError(err)
-    }
-  })
-
   // Restart everything we were torrenting last time the app ran
   resumeTorrents()
 
   // Calling update() updates the UI given the current state
   // Do this at least once a second to give  every file in every torrentSummary
   // a progress bar and to keep the cursor in sync when playing a video
-  setInterval(function () {
-    if (!updateTorrentProgress()) {
-      update() // If we didn't just update(), do so now, for the video cursor
-    }
-  }, 1000)
+  setInterval(update, 1000)
 }
 
 // This is the (mostly) pure function from state -> UI. Returns a virtual DOM
 // tree. Any events, such as button clicks, will turn into calls to dispatch()
 function render (state) {
-  return App(state)
+  try {
+    return App(state)
```

Comment:
Yes.

Now when there's an exception in view code, it blanks out the window and prints the *actual* stack trace which made debugging a lot easier